<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextSecure by zhiyan114</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Encryption Mode</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="EncryptBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EncClear">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Output</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="EncInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Your Data Input Here</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="EncOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Your Data Input Here</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Decryption Mode</string>
    </property>
    <widget class="QPushButton" name="DecryptBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DecClear">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Output</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Message is original: NULL</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="DecInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Your Data Input Here</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="DecOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Your Data Input Here</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="KeyInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="placeholderText">
     <string>Your AES Key Here (16, 24, or 32 characters long)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
